scripts/filter_fastq.py

```diff
@@ -0,0 +1,72 @@
+def avg_quality(seq_quality):
+    """
+    Calculate the average quality score of the given sequence quality.
+
+    Parameters
+    ----------
+    seq_quality : str
+        The sequence quality to calculate the average score for.
+
+    Returns
+    -------
+    float
+        The average quality score of the sequence quality.
+    """
+    return sum(ord(base) - 33 for base in seq_quality) / len(seq_quality)
+
+
+def gc_percent(seq):
+    """
+    Calculate the GC content of the given sequence.
+
+    Parameters
+    ----------
+    seq : str
+        The sequence to calculate the GC content for.
+
+    Returns
+    -------
+    float
+        The GC content of the sequence as a percentage.
+    """
+    return (seq.count("G") + seq.count("C")) / len(seq) * 100
+
+
+def convert_bounds(bounds):
+    """
+    Convert bounds to a valid range.
+    If only one bound is passed, convert it to [0, bounds].
+
+    Parameters
+    ----------
+    bounds : int, float, list
+        The bounds to convert.
+
+    Returns
+    -------
+    list
+        The converted bounds.
+    """
+    if isinstance(bounds, int) or isinstance(bounds, float):
+        return [0, bounds]
```

Comment:
Это кстати можно так сделать:
```suggestion
    """
    if isinstance(bounds, (int, float)):
        return [0, bounds]
```